src/screens/SSIPersonalDataScreen/index.tsx

```diff
@@ -94,7 +94,7 @@ class SSIPersonalDataScreen extends PureComponent<IProps, IState> {
     return (
       <TouchableWithoutFeedback onPress={Keyboard.dismiss} accessible={false}>
         <Container>
-          <SSIScrollView>
+          <SSIScrollView keyboardShouldPersistTaps="handled">
```

Comment:
move this to the styling component

Reply:
moved to `SSIFullHeightScrollViewContainer`